plugins/ruby-foreman-fog-proxmox/foreman_fog_proxmox.rb

```diff
@@ -1,2 +1 @@
-gem 'foreman_fog_proxmox', '0.17.1'
-gem 'fog-proxmox', '0.15.0'
```

Comment:
I don't think this should be removed

Reply:
Why not? I did the same in https://github.com/theforeman/foreman-packaging/pull/11801
The plugin will load the gem itself

Reply:
At first it looked like accidental removal.

> The plugin will load the gem itself

Would that mean that we never needed this or it was needed in the past but not anymore?

Reply:
I lean towards "never needed".
(tasks also doesn't list any deps: https://github.com/theforeman/foreman-packaging/blob/deb/develop/plugins/ruby-foreman-tasks/foreman-tasks.rb)

Reply:
This has been here since https://github.com/theforeman/foreman-packaging/pull/3251 and I see no reason for it.